salesapp.js

```diff
@@ -1,315 +1,442 @@
-// //table
-var tableEL = document.createElement('table');
 
-var stores = [pikePlace, seaTac, seacCenter, capHill, alKai];
-for(var i = 0; i < stores.length; i++){
-  var currentStore = stores[i];
+CookieStore.prototype.sumSales = function(){
+  var total = 0;
+  for ( var i = 0; i < this.hourlyCookies.length; i++ ){
+    total += this.hourlyCookies[i];
+  }
+  this.totalSales = total;
+  return total;
+};
 
+CookieStore.prototype.populateTable = function() {
   var rowEl = document.createElement('tr');
-  var tableEl = appendChild(rowEl);
-
   var nameEl = document.createElement('th');
-  nameEl.textContent = currentStore.name;
-  //<th>currentStore.name</th>
+  nameEl.textContent = this.storeName;
   rowEl.appendChild(nameEl);
+  for( var i = 0; i < this.hourlyCookies.length; i++){
+    var fieldEl = document.createElement('td');
+    fieldEl.textContent = this.hourlyCookies[i];
+    rowEl.appendChild(fieldEl);
+  }
+  var fieldTotal = document.createElement('td');
+  fieldTotal.textContent = this.totalSales;
+  rowEl.appendChild(fieldTotal);
+  tableEl.appendChild(rowEl);
 
-  var minCustEl = document.createElement('td');
-  minCustEl.textContent = currentStore.minHourCustomers;
-  rowEl.appendChild(minCustEl);
+};
 
-  var maxCustEl = document.createElement('td');
-  maxCustEl.textContent = currentStore.maxHourCustomers;
-  rowEl.appendChild(maxCustEl);
 
-  var avgCookiesEl = document.createElement('td');
-  avgCookiesEl.textContent = currentStore.avgCookies;
-  rowEl.appendChild(avgCookiesEl);
-  tableEl.appendChild(rowEl);
+var pikePlace = new CookieStore('pike', 'Pike Place', 23, 65, 6.3);
+var seaTac = new CookieStore('seatac', 'SeaTac', 3, 24, 1.2);
+var seaCenter = new CookieStore('seacent', 'Seattle Center', 11, 38, 3.7);
+var capHill = new CookieStore('caphill', 'Capitol Hill', 20, 38, 2.8);
+var alKai = new CookieStore('alkai', 'Alkai', 2, 16, 4.6);
+var storeArray = [pikePlace, seaTac, seaCenter, capHill, alKai];
+var storeTimes = ['Location', '6:00am', '7:00am', '8:00am', '9:00am', '10:00am', '11:00am', '12:00pm', '1:00pm', '2:00pm', '3:00pm', '4:00pm', '5:00pm', '6:00pm', '7:00pm', '8:00pm', 'Total'];
+
+function runAllStoreData() {
+  for (var i = 0; i < storeArray.length; i++) {
+    storeArray[i].avgSales();
+    storeArray[i].sumSales();
+  }
 };
 
-document.body.appendChild(tableEl);
 
+var tableEl = document.createElement('table');
+
+var sectionEl = document.getElementById('cookie-table');
+sectionEl.appendChild(tableEl);
+
+var runTableTimes = function(){
+var tableTimes = document.createElement('tr');
+for( var i = 0; i < storeTimes.length; i++){
+    var fieldEl = document.createElement('td');
+    fieldEl.textContent = storeTimes[i];
+    tableTimes.appendChild(fieldEl);
+  }
+
+  tableEl.appendChild(tableTimes);
+};
+
+runTableTimes();
+runAllStoreData();
+
+for(var i = 0; i < storeArray.length; i++){
+  storeArray[i].populateTable();
+};
+
+// var tableEL = document.createElement('table');
```

Comment:
You already have all this code preserved in your git history, so if you ever want to see it, you can just go back through your old commits. You don't need to keep it here, commented out for posterity unless you think it's helpful for reference. So, if you aren't using this for reference anymore, feel free to delete it.